src/hotspot/share/utilities/debug.cpp

```diff
@@ -421,8 +421,8 @@ extern "C" JNIEXPORT void nm(intptr_t p) {
   // Actually we look through all CodeBlobs (the nm name has been kept for backwards compatibility)
   Command c("nm");
   CodeBlob* cb = CodeCache::find_blob((address)p);
-  if (cb == NULL) {
-    tty->print_cr("NULL");
+  if (cb == nullptr) {
+    tty->print_cr("nullptr");
```

Comment:
Changes log, OK?

Reply:
I think it's ok. Unless we have tests that check for NULL, but they're likely in gtests and you'll find them pretty quickly and can fix them pretty quickly if they slip through.